src/alr/alr-commands-init.adb

```diff
@@ -43,12 +43,15 @@ package body Alr.Commands.Init is
       end if;
 
       declare
-         Guard : Folder_Guard (
```

Comment:
Removing this one causes a spurious 'alire' folder to be created in the CWD. Possible fix is to change line 163 from 
`OS_Lib.Create_Folder (Paths.Alr_Working_Folder);` 
to 
`OS_Lib.Create_Folder (Name / Paths.Alr_Working_Folder);`
(without undoing the deletion)

Reply:
Thank you for catching this! I’ll have a look.